packages/odyssey-react-mui/src/DataTable/DataTable.tsx

```diff
@@ -377,6 +381,7 @@ const DataTable = ({
   noResultsPlaceholder,
   filters: filtersProp,
   renderDetailPanel,
+  initialSearchValue = "",
```

Comment:
@jordankoschei-okta Can we alphabetize props here. And in the type definition as well 😄